Sources/Tokamak/Components/Props/Style.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Max Desiatov on 31/12/2018.
 //
 
-public protocol StyleProps {
+public protocol ViewProps {
```

Comment:
as `accessibility` is a property of `Style` after all, I think this should be renamed back to `StyleProps` to leave its use as clear as it was before.